via/static/scripts/video_player/components/Transcript.tsx

```diff
@@ -194,6 +200,20 @@ function TranscriptSegment({
         )}
         data-testid="transcript-text"
         ref={contentRef}
+        // We have a "click" handler here, but don't set a role because
+        // this is a secondary way to focus the transcript. The timestamp
+        // button is the primary control for this action.
```

Comment:
The result I'm going for here is that the segment timestamp is the accessible way of seeking the video - a button with completely ordinary behavior. Clicking the text is an alternative method, but not marked as a button because it has some non-standard behavior associated with it.